plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -3619,3 +3619,96 @@ func TestVolumeZonePredicate(t *testing.T) {
 
 	}
 }
+
+func TestVolumeZonePredicateMultiZone(t *testing.T) {
+	pvInfo := FakePersistentVolumeInfo{
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "Vol_1", Labels: map[string]string{kubeletapis.LabelZoneFailureDomain: "zone_1"}},
+		},
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "Vol_2", Labels: map[string]string{kubeletapis.LabelZoneFailureDomain: "zone_2", "uselessLabel": "none"}},
+		},
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "Vol_3", Labels: map[string]string{kubeletapis.LabelZoneFailureDomain: "zone_3, zone_1"}},
```

Comment:
Is "zone_3, zone_1" a valid label value in Kubernetes?

Reply:
No, I just hit that in my testing. The value of a label can not contain commas or spaces, per [documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set): label values can only `consist of alphanumeric characters, '-', '_' or '.'`.

Which means we'll have to use a different delimiter here. 

Since both AWS and GCE zones are alphanumeric and `-`. I'm going to use two underscores, e.g. `__` as the delimiter. So a label would look like `failure-domain.beta.kubernetes.io/zone=us-west1-a__us-west1-b__us-west1-c`. Not pretty, but it works, and even if some future cloud starts using underscores in zone names, two back to back underscores should be unlikely.

This should be sufficient for now to expand the beta zone failure domain functionality to support RePD. We can consider other solutions (maybe having one key per zone, e.g. `zone.failure-domain.kubernetes.io/us-west1-a=true`), as part of the end to end design for handling topology in Kubernetes.

Reply:
Two underscores SGTM for this PR, will you update the code and test to make it effect? And we may need to fire a issue to discuss the further improvement like `one key per zone` or other solution.

Reply:
Yep, made the change, testing it now, will send out an update shortly. And I'm planning on submitting a topology proposal to the community repo with the options and we can use that to further discuss these issues.